stock_inventory_discrepancy/wizards/confirm_discrepancy_wiz.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="stock_quant_confirm_discrepancy" model="ir.ui.view">
+        <field name="model">stock.quant</field>
+        <field name="inherit_id" ref="stock.view_stock_quant_tree_inventory_editable" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <button name="action_apply_inventory" position="attributes" />
+            <button name="action_set_inventory_quantity_to_zero" position="replace" />
```

Comment:
Using replace you must add priority > 98 in this view

Reply:
I'd prefer setting it as invisible.